slam-ws/src/nav_husky_gazebo/config/global_costmap_params.yaml

```diff
@@ -1,7 +1,7 @@
 global_costmap:
-  global_frame: /odom   # Because we don't have a map to load in, we use the odom frame as our global frame (rather than /map)
+  global_frame: /odom   
```

Comment:
Keep this comment (maybe rephrase a bit) to explain why `odom` is used when no static map is supplied